privacy-on-beam/pbeam/count_test.go

```diff
@@ -67,6 +67,40 @@ func TestCountNoNoise(t *testing.T) {
 	}
 }
 
+// Checks that Count with partitions returns a correct answer.
+func TestCountWithPartitionsNoNoise(t *testing.T) {
+	var pairs []pairII
+	for i := 0; i < 10; i++ {
+		v := makePairsWithFixedVStartingFromKey(i, 1, i)
+		pairs = append(pairs, v...)
+	}
+
+	var result []testInt64Metric
+	for i := 9; i < 11; i++ {
+		result = append(result, testInt64Metric{i, 1})
+	}
+
+	p, s, col, want := ptest.CreateList2(pairs, result)
+	col = beam.ParDo(s, pairToKV, col)
+	partitions := []int{}
+	for i := 9; i < 11; i++ {
+		partitions = append(partitions, i)
+	}
+
+	partitionsCol := CreateList(s, partitions)
+
```

Comment:
I'm not sure if flakiness matters here, because partitions are specified. What do you think?

Reply:
It still applies because we are adding noise to the aggregation, i.e. the counts. If we don't use a flakiness here, there is a very small chance it might add enough noise to change the output. Does that make sense?